server/channels/app/platform/log.go

```diff
@@ -206,6 +207,40 @@ func (ps *PlatformService) GetLogsSkipSend(page, perPage int, logFilter *model.L
 	return lines, nil
 }
 
+func (ps *PlatformService) GetLogFile() (*model.FileData, error) {
```

Comment:
I moved the code here so both of the implementations can access it. I even thought about moving the whole Support Packet generation into the platform. What do you think?

Reply:
the migration process might introduce some initial overhead, but I think it is worth it for the long term benefits, maybe open a new ticket for it.

Reply:
Let's keep the idea of moving the Support Packet generation code in a shared place in mind. I even thought about introducing a platform service for the Support Packet.